Sources/Workspace/Workspace+Manifests.swift

```diff
@@ -141,9 +141,15 @@ extension Workspace {
                 let dependency = dependency.dependency
                 switch dependency.state {
                 case .sourceControlCheckout(let checkout):
-                    if checkout.isBranchOrRevisionBased {
-                        result.insert(dependency.packageRef)
+                    let packageRef = dependency.packageRef
+
+                    if checkout.isBranchOrRevisionBased ||
+                       // FIXME: Remove this once we have a general mechanism
+                       //        for passing "safe" flags.
+                       packageRef.identity == .plain("swift-corelibs-foundation") {
```

Comment:
Formatting looks off for this snippet, unless GitHub is mangling the diff? We stick to 4 spaces, you can also run [SwiftFormat](https://github.com/nicklockwood/SwiftFormat) (`brew install swiftformat`) on this file to automate that.

Reply:
I keep forgetting to ask about this, thank you!

Reply:
✅